src/main/scala/org/apache/spark/sql/delta/commands/DeltaCommand.scala

```diff
@@ -37,7 +37,7 @@ trait DeltaCommand extends DeltaLogging {
    *
    * @throws AnalysisException if a non-partition column is referenced.
    */
-  protected def parsePartitionPredicates(
+  protected def parsePredicates(
```

Comment:
Renamed to better match the reality that it parses all predicates and not only partitions.